meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx6ullevk/lmp.cfg

```diff
@@ -1,10 +1,13 @@
 CONFIG_SPL=y
+# MAX SIZE = D000(SPL+dtb) + 2060(csf) - 400(fit signing data)
+CONFIG_SPL_SIZE_LIMIT=0xEC60
 CONFIG_SPL_SEPARATE_BSS=y
 CONFIG_SPL_LDSCRIPT="arch/arm/mach-omap2/u-boot-spl.lds"
 CONFIG_SPL_TEXT_BASE=0x00908000
-CONFIG_SPL_FIT_SIGNATURE=y
-CONFIG_SPL_FIT_SIGNATURE_STRICT=y
+CONFIG_SPL_FIT_SIGNATURE=n
+CONFIG_SPL_FIT_SIGNATURE_STRICT=n
```

Comment:
So we disable signature verification of FIT image in mfgtool U-Boot?

BTW:
```
CONFIG_SPL_FIT_SIGNATURE is not set
CONFIG_SPL_FIT_SIGNATURE_STRICT is not set
```

Reply:
This may need some documentation to explain why the disable is here

Reply:
Right, we need this enabled as well otherwise the SPL used by mfgtools can later be used to boot any image that is not signed.

Guess we need some size optimizations for being able to enable this here.

Reply:
Yes, this is a temporary size optimization so that we can sign the mfgtool SPL on a closed board.  I wasn't sure how to support signing using the RAM load method.  This needs to be debugged and fixed.

Reply:
@igoropaniuk Regarding the specific "=n" form here.  I wanted this to stick out and I will add a comment above it regarding the size optimization TODO.